internal/component/prometheus/exporter/oracledb/oracledb.go

```diff
@@ -48,6 +47,7 @@ type Arguments struct {
 	MaxIdleConns     int               `alloy:"max_idle_conns,attr,optional"`
 	MaxOpenConns     int               `alloy:"max_open_conns,attr,optional"`
 	QueryTimeout     int               `alloy:"query_timeout,attr,optional"`
+	CustomMetrics      alloytypes.OptionalSecret `alloy:"custom_metrics,attr,optional"`
```

Comment:
Do you expect this to contain sensitive data such as passwords, etc? Could it be a `string` type instead?

Reply:
i think as a file path not needed, will change to string !